src/wpantund/wpan-properties.h

```diff
@@ -48,6 +48,7 @@
 #define kWPANTUNDProperty_DaemonAutoDeepSleep                   "Daemon:AutoDeepSleep"
 #define kWPANTUNDProperty_DaemonFaultReason                     "Daemon:FaultReason"
 #define kWPANTUNDProperty_DaemonSetDefRouteForAutoAddedPrefix   "Daemon:SetDefaultRouteForAutoAddedPrefix"
+#define kWPANTUNDProperty_DaemonSetSLAACForAutoAddedPrefix      "Daemon:SetSLAACForAutoAddedPrefix"
```

Comment:
It feels like we should use a different property category instead of `Daemon:`. How about `IPv6:`?

I realize @abtink set a precedent with line 50 (from #209), but I'm not sure it is the right one. @abtink, feel free to chime in with your thoughts.

Reply:
I am fine with changing them both to `"IPv6:<name>"`